test/components/portal.spec.tsx

```diff
@@ -1,36 +1,38 @@
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
-import {ClientRenderer, expect, selectDom, waitFor} from 'test-drive-react';
+import {ClientRenderer, expect, waitFor} from 'test-drive-react';
 import {Portal} from '../../src';
-
-const portalId = 'PORTAL';
+import {PortalTestDriver} from '../../test-kit/components/portal-driver';
```

Comment:
add index to components dir and to test-kit dir.
import from test-kit

Reply:
done